### PR TITLE
[Blog Post] Added a new blog post.Technical article from K8S blog.

### PR DESCRIPTION
**Pull request by anvithks**
* **Please check if the PR fulfills these requirements**
- [x] The commit message follows our guidelines

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
New blog post.  
KubeEdge was accepted into the CNCF Sandbox. This blog post is an announcement on the website.  
Referenced the blog post on the Kubernetes website.  

* **Other information**:

### CODE REVIEW

**Comment by m1093782566** [comment]
To be clear, it is not the CNCF announcement, it is the technical article posted in Kubernetes blog.

**Comment by anvithks** [comment]
@m1093782566 Yes. I was referring to this post also having the announcement.
I will edit the pr title.

**Comment by m1093782566** [comment]
@anvithks 

I think you misunderstand it. The blog in Kubernetes website is just a technical article introducing kubeedge and it's not the announcement made by CNCF.

**Comment by anvithks** [comment]
@m1093782566 Correct. I understand that. 
What I meant was that this blog post also has the announcement regarding the sandboxing.
I have updated the PR title now. Please take a look.

File: content/zh/blog/cncf-sandbox-announcement/index.md [hunk 1]
```diff
@@ -0,0 +1,139 @@
++++
+title = "KubeEdge has been accepted into the CNCF Sandbox"
```
Comment (m1093782566) [inline content/zh/blog/cncf-sandbox-announcement/index.md]: The title is inconsistent with the content and English article should not be in `content/zh`.

File: content/en/blog/cncf-sandbox-announcement/index.md [hunk 2]
```diff
@@ -0,0 +1,139 @@
++++
+title = "KubeEdge has been accepted into the CNCF Sandbox"
```
Comment (m1093782566) [inline content/en/blog/cncf-sandbox-announcement/index.md]: The title is inconsistent with the content and this article has nothing to do with cncf sandbox announcement.

File: content/zh/blog/cncf-sandbox-announcement/index.md (hunk 1 above)
Comment (anvithks) [reply content/zh/blog/cncf-sandbox-announcement/index.md]: Updated. Please a take a look.

File: content/en/blog/cncf-sandbox-announcement/index.md (hunk 2 above)
Comment (anvithks) [reply content/en/blog/cncf-sandbox-announcement/index.md]: Updated. Please a take a look.

**Pull request merged**